### ISSUE TITLE
Make docker image which runs node 8.10 for deployments & build

### ISSUE THREAD

**Issue by benswinburne** [guyfedwards/emojo#17]
The yarn install downloads and compiles binaries needed for sharp and gifcicle so it  needs to be the same as the one which it runs on it seems.

https://github.com/lovell/sharp/issues/1258

When deploying from osx the lambda function 

```
module initialization error: Error
at Object.hasVendoredLibvips (/var/task/node_modules/sharp/lib/libvips.js:61:13)
at Object.<anonymous> (/var/task/node_modules/sharp/lib/constructor.js:9:22)
at Module._compile (module.js:652:30)
at Object.Module._extensions..js (module.js:663:10)
at Module.load (module.js:565:32)
at tryModuleLoad (module.js:505:12)
at Function.Module._load (module.js:497:3)
at Module.require (module.js:596:17)
at require (internal/module.js:11:18)
at Object.<anonymous> (/var/task/node_modules/sharp/lib/index.js:3:15)
```

Look at this

https://github.com/adieuadieu/retinal/tree/master/lambda-sharp

Would be good to get something like this going for ourselves

**Comment by guyfedwards** [comment]
Interesting. I had this issue locally the other day and solved by re-installing deps. Will take a look at 'dockerizing' stuff

**Comment by benswinburne** [comment]
Closed by #27
